examples/package.json

```diff
@@ -41,6 +42,7 @@
     "chromadb": "^1.4.0",
     "faiss-node": "^0.2.1",
     "graphql": "^16.6.0",
+    "hdb-pool": "^0.1.6",
```

Comment:
Where are these required?

Reply:
Good catch. Yes, hdb-pool is implicitly required by `typeorm` even though the package has been deprecated for a while ([hdb-pool github](https://github.com/ckyycc/hdb-pool)) since the features are now supported by NodeJs natively.

`typeorm` allows you to specify your own driver in the connection settings (https://github.com/typeorm/typeorm/blob/master/src/driver/sap/SapDriver.ts#L818), but doing so breaks the code as it is written for hdb-pool only, e.g. expecting an `eventEmitter` property in the driver: (https://github.com/typeorm/typeorm/blob/master/src/driver/sap/SapDriver.ts#L287).

So the `hdb-pool` dependency is needed until `typeorm` updates their code.

Reply:
Got it - since it's not explicitly in the code, let's not have this in the package.json here and add it to the docs instead.